sdcm/cluster.py

```diff
@@ -4369,6 +4369,14 @@ def node_setup(self, node: BaseNode, verbose: bool = False, timeout: int = 3600)
         if self.params.get('logs_transport') == 'ssh':
             node.install_package('python3')
 
+        if node.is_ubuntu():
+            result = node.remoter.sudo("pro status", ignore_status=True)
```

Comment:
are we should this command is available on other ubuntu ? did we tested it on any other ubuntu artifact test ?

Reply:
It should be according to what I understand from the website, but I'll run the ubuntu 20 and 22 artifacts to be sure

Reply:
[Done](https://jenkins.scylladb.com/job/scylla-staging/job/Shlomo/job/artifacts-ubuntu2004-test-FIPS-branch/2)
and
[Done](https://jenkins.scylladb.com/job/scylla-staging/job/Shlomo/job/artifacts-ubuntu2204-test-FPIS-brnach/2)

Reply:
No